### PR TITLE
Featured recipes

### PR DESCRIPTION
**Pull request by rohansree**
Added functionality that displays random recipe cards on the home page. Needs design work, but functionality works.

### CODE REVIEW

File: source/assets/scripts/main.js [hunk 1]
```diff
@@ -203,6 +214,77 @@ async function fetchRandomAPI() {
  *                      RECIPE CARDS
  ****************************************************************************/
 
+/**
+ * function binds the Featured Recipe Cards to the homepage
```
Comment (dwk002) [inline source/assets/scripts/main.js:218]: Soon, we'll have to change this comment to be compatible with automatic documenting with `JSDoc`

File: source/assets/scripts/main.js [hunk 2]
```diff
@@ -203,6 +214,77 @@ async function fetchRandomAPI() {
...
+ * and binds the Featured Recipe Cards to recipe viewer.
```
Comment (dwk002) [inline source/assets/scripts/main.js:219]: Awesome Function!!!!

**Review by dwk002 — APPROVED** [review]

File: source/assets/scripts/main.js (hunk 1 above)
Comment (rohansree) [reply source/assets/scripts/main.js:218]: There aren't any parameters or return types. Does it need anything else to be JSDoc compatible?

Comment (dwk002) [reply source/assets/scripts/main.js:218]: Wait a second, you totally right. My bad~

**Pull request merged**